x-pack/heartbeat/monitors/browser/browser.go

```diff
@@ -11,28 +11,28 @@ import (
 	"os/user"
 	"sync"
 
-	"github.com/elastic/beats/v7/heartbeat/monitors"
 	"github.com/elastic/beats/v7/heartbeat/monitors/jobs"
+	"github.com/elastic/beats/v7/heartbeat/monitors/plugin"
+	"github.com/elastic/beats/v7/libbeat/beat"
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/elastic/beats/v7/x-pack/heartbeat/monitors/browser/synthexec"
 )
 
 func init() {
-	monitors.RegisterActive("browser", create)
-	monitors.RegisterActive("synthetic/browser", create)
+	plugin.Register("browser", create, "synthetic", "synthetics/synthetic")
```

Comment:
Why the rename form `synthetics/browser` to just `synthetics/synthetic`? That just seems like weird naming.

Reply:
In the future we'd like to have a `synthetics/api`, and we use the browser language across the board from UI to backend